sycl/plugins/esimd_emulator/pi_esimd_emulator.hpp

```diff
@@ -105,6 +105,27 @@ struct _pi_queue : _pi_object {
   cm_support::CmQueue *CmQueuePtr = nullptr;
 };
 
+struct cm_surface_ptr_t {
+  // 'UP' means 'User-Provided' in CM Lib - corresponding to
+  // Buffer/Image created with PI_MEM_FLAGS_HOST_PTR_USE option in
+  // SYCL
+  enum SurfaceType {
+    type_none,
```

Comment:
Stylistical question: I see most identifiers are using camel case here, but these do not. Is it intended?

Reply:
It was based on Konst's initial suggestion. I agree that they look inconsistent.

Reply:
Camel case type is applied.